specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/gallery.json

```diff
@@ -2098,6 +2098,16 @@
         "manageActions": {
           "$ref": "#/definitions/UserArtifactManage"
         },
+        "settings": {
+          "$ref": "#/definitions/UserArtifactSettings"
+        },
+        "advancedSettings": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Optional. Additional settings to pass to the VMApp extension. For advanced used only."
```

Comment:
'For advanced used only' what do you mean?
Should it be 'use'?

Reply:
Changed to "For advanced use only"